### PR TITLE
Implment adaptors in AVM, and TEE operators should be supported in fns:v2

### PR DESCRIPTION
**Pull request by Weber-H2O2**

### CODE REVIEW

**Comment by Weber-H2O2** [comment]
@eigmax Update according to the merged code. Please review.

File: packages/arb-avm-cpp/avm/src/machinestate/machineoperation.cpp [hunk 1]
```diff
@@ -1007,13 +1003,110 @@ void ecall(MachineState& m) {
...
+        std::cerr << "[TEESDK] submit_task output " << "[" << outputsize << "]: " << output << std::endl;
+
+        // Success
+        out_arg1 = 0;
+
+        switch (intx::narrow_cast<std::size_t>(arg1)) {
+        case 0:
+            // TODO
+            break;
+        case 1: // Add
```
Comment (eigmax) [inline packages/arb-avm-cpp/avm/src/machinestate/machineoperation.cpp:1081]: Sync the comment with TEE?

File: packages/arb-avm-cpp/avm/src/machinestate/machineoperation.cpp [hunk 2]
```diff
@@ -987,8 +985,6 @@ void ecall(MachineState& m) {
     const char *pub = getenv("TEESDK_PUB");
     const char *pri = getenv("TEESDK_PRI");
     const char *conf = getenv("TEESDK_CONF");
-    const char *method = getenv("TEESDK_METHOD");
-    const char *args = getenv("TEESDK_ARGS");
     const char *uid = getenv("TEESDK_UID");
     const char *token = getenv("TEESDK_TOKEN");
```
Comment (eigmax) [inline packages/arb-avm-cpp/avm/src/machinestate/machineoperation.cpp:989]: Do fast failure when env is wrongly set?

File: packages/arb-avm-cpp/avm/src/machinestate/machineoperation.cpp (hunk 1 above)
Comment (Weber-H2O2) [reply packages/arb-avm-cpp/avm/src/machinestate/machineoperation.cpp]: OK, I've fixed these comments in the latest commit.

File: packages/arb-avm-cpp/avm/src/machinestate/machineoperation.cpp (hunk 2 above)
Comment (Weber-H2O2) [reply packages/arb-avm-cpp/avm/src/machinestate/machineoperation.cpp:995]: You're right. I've fixed these comments in the latest commit.

**Review by eigmax — APPROVED** [review]
LGTM

**Pull request merged**